src/app/tests/suites/certification/Test_TC_BINFO_2_1.yaml

```diff
@@ -715,3 +714,87 @@ tests:
       attribute: "ProductAppearance"
       response:
           value: ProductAppearancevalue
+
+    - label: "Step 62: TH reads SpecificationVersion from the DUT."
+      PICS: BINFO.S.A0015
+      command: "readAttribute"
+      attribute: "SpecificationVersion"
+      response:
+          saveAs: SpecificationVersionValue
+
+    - label:
+          "Step 62: If the SpecificationVersion value is absent or zero this
+          step cannot be verified, move on to the next step"
+      PICS: BINFO.S.A0015
+      cluster: "EqualityCommands"
+      command: "UnsignedNumberEquals"
+      arguments:
+          values:
+              - name: "Value1"
+                value: SpecificationVersionValue
+              - name: "Value2"
+                value: null
+      response:
+          - values:
+                - name: "Equals"
+                  saveAs: IsExpectedValue
+
+    - label:
+          "Step 62: SpecificationVersion value should in the inclusive range of
+          (0 to 4294967295) and the initial 7 bits are set to zero."
+      runIf: "!IsExpectedValue"
+      PICS: BINFO.S.A0015
+      command: "readAttribute"
+      attribute: "SpecificationVersion"
+      response:
+          value: SpecificationVersionValue
+          constraints:
+              minValue: 0
+              maxValue: 4294967295
+              hasMasksClear: [0x1, 0x2, 0x4, 0x8, 0x10, 0x20, 0x40]
+
+    - label:
+          "Step 63: TH sends Write request message to DUT to change the value of
+          SpecificationVersion to '0x0103AAF1'."
+      PICS: BINFO.S.A0015
+      command: "writeAttribute"
+      attribute: "SpecificationVersion"
+      arguments:
+          value: "0x0103AAF1"
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "Step 64: TH reads SpecificationVersion attribute from DUT"
+      PICS: BINFO.S.A0015
+      command: "readAttribute"
+      attribute: "SpecificationVersion"
+      response:
+          value: SpecificationVersionValue
+
+    - label: "Step 65: TH reads MaxPathsPerInvoke from the DUT."
+      PICS: BINFO.S.A0016
+      command: "readAttribute"
+      attribute: "MaxPathsPerInvoke"
+      response:
+          saveAs: MaxPathsPerInvokeValue
+          constraints:
+              minValue: 1
+              maxValue: 65535
+
+    - label:
+          "Step 66: TH sends Write request message to DUT to change
+          MaxPathsPerInvoke to a different value."
+      PICS: BINFO.S.A0016
+      command: "writeAttribute"
+      attribute: "MaxPathsPerInvoke"
+      arguments:
+          value: 12345
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "Step 67: TH reads MaxPathsPerInvoke attribute from DUT"
```

Comment:
Likely outside the scope of this PR, but are there any csg test that confirm that SessionParameters match basic info attributes match? Likely this would need to be a python tests and yaml doesn't expose session parameters

@tehampson (I mention myself, as this triggers filters for myself so that I get notification)